weblate/trans/tasks.py

```diff
@@ -228,8 +227,14 @@ def cleanup_suggestions():
 def update_remotes():
     """Update all remote branches (without attempt to merge)."""
     non_linked = Component.objects.with_repo()
-    for component in non_linked.iterator():
-        perform_update.delay("Component", component.pk, auto=True)
+    if settings.AUTO_UPDATE == "none":
+        pass
+    elif settings.AUTO_UPDATE not in ("none", "full", "remote", True, False):
```

Comment:
I'd probably simplify all this to just:

```suggestion
    if settings.AUTO_UPDATE not in ("full", "remote", True, False):
        return
```

Reply:
thanks for suggestion